src/pages/VoucherRedemption/VoucherRedemption.tsx

```diff
@@ -28,6 +29,34 @@ const VoucherRedemption = (props: Props) => {
   const [loading, setLoading] = useState(false);
 
   const fetchData = async () => {
+    let getLocationInfo = (merchantData): LocationInfo => {
+      const emptyLocationInfo = {
+        line1: '',
+        line2: '',
+      };
+      if (!merchantData || !merchantData.data) {
+        return emptyLocationInfo;
+      }
+      const locations = merchantData.data.locations;
+      if (!locations || !locations.length) {
+        return emptyLocationInfo;
+      }
+      const location = locations[0];
+      if (!location) {
+        return emptyLocationInfo;
+      }
+      const locationInfo = {
+        line1: (location.address1 ?? '') + ' ' + (location.address2 ?? ''),
```

Comment:
Could you use JavaScript string interpolation here?

Reply:
Good call!